docs/index.bs

```diff
@@ -2262,7 +2260,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
     The user agent *must* create a {{CacheStorage}} object when a {{Window}} object or a {{WorkerGlobalScope}} object is created and associate it with that <dfn for="CacheStorage">global object</dfn>.
 
-    A {{CacheStorage}} object represents a <a>name to cache map</a> of its associated [=CacheStorage/global object=]'s <a>environment settings object</a>'s [=environment settings object/origin=]. Multiple separate objects implementing the {{CacheStorage}} interface across documents and workers can all be associated with the same <a>name to cache map</a> simultaneously.
+    A {{CacheStorage}} object represents the [=name to cache map=] associated with its associated [=CacheStorage/global object=]'s [=environment settings object=]. Multiple separate objects implementing the {{CacheStorage}} interface across documents and workers can all be associated with the same <a>name to cache map</a> simultaneously.
```

Comment:
using def ref syntax for <a>name to cache map</a> to [=name to cache map=] and also maintains consistency

Reply:
done.